lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7903,9 +7903,39 @@ createUnavailableDecl(Identifier name, DeclContext *dc, Type type,
   return var;
 }
 
+static ExtensionDecl *figureOutTheDeclarationContextToImportInto(Decl* D, DeclContext *&DC, IterableDeclContext *&IDC) {
+  if (auto nominal = dyn_cast<NominalTypeDecl>(D)) {
+    DC = nominal;
+    IDC = nominal;
+    return nullptr;
```

Comment:
Using a return and then an else is against llvm style. You already have the control flow you need, so please remove the else.

https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return